data/vetter2020daily/vetter2020daily

```diff
@@ -0,0 +1,128 @@
+#import modules;
+import os
+import yaml
+import pandas as pd
+import numpy as np
+
+#We won't need this when making markdown file;
+os.chdir("./data/vetter2020daily")
+
+#functions to add folded blocks and literal blocks;
+class folded_str(str): 
+    pass
+class literal_str(str): 
+    pass
+
+def folded_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+def literal_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer)
+
+#read the data;
+df_1 = pd.read_csv("msphere.00827-20-st002.csv")
+
+#The patient information was obtained from the Challenger et al. (2022)
+patient_info = {
```

Comment:
Please update the age based on Table 1 in Vetter et al. (2020), https://journals.asm.org/doi/10.1128/msphere.00827-20. In the markdown file, please explain this either in text blocks or comment in Python code block.